QUARKUS-874.md

```diff
@@ -0,0 +1,37 @@
+# QUARKUS-874 Production support for Windows
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-874
+
+The goal is to ensure some level of support and identify existing limitations on Quarkus side for Windows.
+
+## Scope of the testing
+The selected Test Suites will be executed on the selected Windows Platforms to ensure functionality is fully accomplished.
+In order to accommodate this in our current test suites, we need to rename existing jobs in Jenkins in order to differentiate them for those running on RHEL and on Windows platforms. 
+
+### Selected Windows Platforms
+- Windows Server 2019
+
+### Selected Test Suites
+ - Start-Stop Test Suite
+   - In Github CI - for Pull Requests
+   - New Jenkins Jobs
+ - Beefy Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+   - New Jenkins Jobs
+ - QuickStarts Test Suite
+   - New Jenkins Jobs
+ - Combinations of Quarkus extensions Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+ - Workshop Test Suite
+   - New Jenkins Jobs
+
+### Platform testing
+Tests are going to be executed against all tested versions of databases, RHEL and OpenShift versions. 
+Both the JVM and native mode is going to be tested.
+
+## Impact on resources
+- New Windows instances in our Jenkins environments
+  - Windows instance must support Native compilation (GraalVM, VS Build Tools) and Docker (Windows Containers)
```

Comment:
Native compilation (GraalVM, VS Build Tools) is not required by this RFE, I would drop it from this text.
I want that to be available so we have option to try native binaries on Windows eventhough it's not this RFE requirement.

Docker (Windows Containers) is not what we want. We need Docker with Linux containers support.
Our testsuites rely on testcontainers to assure portability between different CI environments and to avoid dependency on many external services.

I miss impact on PSI capacity and execution time of our tests.
 - Windows instances will be executed next to the RHEL instances => more CPU / memory in the lab would be needed to ensure similar execution time as now.
 - We do not have new capacity in the lab => test execution will be prolonged (doubled in the worse case)
 - Windows virtual machines usually require more resources than RHEL ones, expecting `large` or `xlarge` flavor to be used
 - More jobs pulling artifacts from Artifactory instance => we may need to use more resources for Artifactory

Reply:
Fixed